src/ui/public/agg_response/_embedded_tooltip.html

```diff
@@ -0,0 +1,7 @@
+<visualization
```

Comment:
have you tried using `<visualize>` ? as you are just loading saved visualizations that might be preferable

Reply:
Not sure that I can use the `<visualize>` directive. How would I update the search source and trigger a search with the visualize directive?

Reply:
you need to pass custom appState object to tooltip instances of visualize. this way they won't observe the dashboard context as normal visualizations do (which are tied to our global appState)

then you can also push filters down to visualization using this appState.